torch/_utils_internal.py

```diff
@@ -159,6 +159,10 @@ def export_training_ir_rollout_check() -> bool:
     return True
 
 
+def full_aoti_runtime_assert() -> bool:
```

Comment:
is this meant to be a config cotnrolled? its always true?

Reply:
> is this meant to be a config controlled? its always true?

it's always True in OSS, internally, we have an env variable and a justknob to control it. Please see the internal Diff for those.